pkg/apis/delivery/v1alpha1/policy_state_types.go

```diff
@@ -45,17 +48,40 @@ var (
 	_ duckv1.KRShaped = (*PolicyState)(nil)
 )
 
-// PolicyStateSpec holds the desired state of the PolicyState
-// Should be set by reconciler, and used by webhook to write Route appropriately
+const (
+	// PolicyStateConditionRouteConfigured is set to false if any failure prevents
+	// PolicyState.Spec from being written to Route.Spec
+	PolicyStateConditionRouteConfigured apis.ConditionType = "RouteConfigured"
+)
+
+// TrafficTarget reuses Route TrafficTarget
+type TrafficTarget v1.TrafficTarget
+
+// PolicyStateSpec holds the desired routing spec computed by reconciler
+// Should be set by reconciler, and set by webhook to write Route appropriately
 type PolicyStateSpec struct {
-	// TODO: implement policy state spec
+	// Traffic specifies how to distribute traffic over a collection of
+	// revisions and configurations.
+	Traffic []TrafficTarget `json:"traffic,omitempty"`
```

Comment:
nit: can this just be `v1.TrafficTarget` instead of having an internal `TrafficTarget`?

Reply:
fixed